web/src/app/requested-items/all/page.tsx

```diff
@@ -1,24 +1,23 @@
-"use client"
+"use client";
 
-import React from 'react'
-import { Box } from '@mui/material'
-import CustomTable from '@/components/Table'
-import { useHook } from './hook'
+import React from "react";
+import { Box } from "@mui/material";
+import CustomTable from "@/components/Table";
+import { useHook } from "./hook";
 
 const AllPage = () => {
+	const { dataSource, columnSchema } = useHook();
+	return (
+		<>
```

Comment:
delete na ni kay single component ra ang sulod
```suggestion
		<>
```